src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -1,15 +1,16 @@
 <template>
-	<div class="tw-flex">
-		<div id="mobile-menu" class="tw-rounded md:tw-hidden
-				tw-z-dropdown tw-drop-shadow tw-absolute tw-top-4 md:tw-top-0"
-		>
-			<loan-search-filter />
-		</div>
+	<div class="tw-flex" @keyup.esc="closeLightbox">
```

Comment:
The `@keyup` portion of this shouldn't be needed as it's built in to the component